onnxruntime/core/mlas/lib/platform.cpp

```diff
@@ -464,3 +465,17 @@
     return MLAS_DEFAULT_PREFERRED_BUFFER_ALIGNMENT;
 #endif
 }
+
+#ifdef MLAS_TARGET_AMD64_IX86
+
+bool
+MLASCALL
+MlasPlatformU8S8Overflow(
+    void
+    )
+{
+    const auto& p = GetMlasPlatform();
+    return p.GemmU8U8Dispatch != p.GemmU8S8Dispatch;
```

Comment:
They are different on SSE 4.1 too, but there is no overflow.

Reply:
SSE4.1 does have overflow problem, it uses _mm_maddubs_epi16, which is [pmaddubsw](https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_maddubs_epi16&ig_expand=4491)